segmentstore/server/host/src/test/java/io/pravega/segmentstore/server/host/handler/PravegaConnectionListenerTest.java

```diff
@@ -195,4 +198,29 @@ public void testCreateRequestProcessor() {
                 mock(StreamSegmentStore.class), mock(TableStore.class), NoOpScheduledExecutor.get());
         Assert.assertTrue(listener.createRequestProcessor(new TrackedConnection(new ServerConnectionInboundHandler())) instanceof AppendProcessor);
     }
+
+    // Test the health status created with pravega listener.
+    @Test
+    public void testHealth() {
+        int port = TestUtils.getAvailableListenPort();
+        PravegaConnectionListener listener = new PravegaConnectionListener(false, port,
+                mock(StreamSegmentStore.class), mock(TableStore.class), NoOpScheduledExecutor.get());
+        listener.startListening();
+        try {
```

Comment:
How is this relevant to the test?

Reply:
I guess we dont need that portion to test if port is available, since its being done earlier. Also, I did modify the test since now the contributor is private.